low-level/files/opt/seagate/sspl/setup/resource_map/server.py

```diff
@@ -214,6 +215,56 @@ def get_platform_sensors_info(self):
                 )
         return response
 
+    def get_sas_hba_info(self):
+        """Return SAS-HBA current health."""
+        sas_hba_data = []
+        hosts = self.sysfs.get_sas_host_list()  # ['host1']
+        for host in hosts:
+            host_id = SAS_RESOURCE_ID + host.replace('host', '')
+            host_data = self.get_health_template(host_id, False)
```

Comment:
Ensure this follows health schema. 
I don't see "last_updated" attribute added to this data.

Reply:
yes added in set_health_data() method, so that each time after setting the final resource health we will set the time.